Dockerfile

```diff
@@ -6,15 +6,19 @@ LABEL org.opencontainers.image.authors="Donghee Na"
 LABEL org.opencontainers.image.title="autoconf 2.72 container for CPython"
 LABEL org.opencontainers.image.description="Container image with autoconf 2.72 tools to regenerate Python's configure script."
 
-RUN apk upgrade
-RUN apk add autoconf automake autoconf-archive libtool pkgconfig alpine-sdk
-RUN wget https://ftp.gnu.org/gnu/autoconf/autoconf-2.72.tar.gz
-RUN tar -zxvf autoconf-2.72.tar.gz
+RUN apk upgrade && \
+    apk add \
+        alpine-sdk \
+        autoconf \
+        automake \
+        autoconf-archive \
+        libtool \
+        pkgconfig
```

Comment:
See https://docs.docker.com/develop/develop-images/instructions/#apt-get

Why is `pkgconfig` and `alpine-sdk` required? Is it because we build GNU Autoconf from scratch later?

Reply:
Yes :)